sonar-go-checks/src/test/resources/checks/DuplicatedFunctionImplementationCheck/duplicated_function_implementation.go

```diff
@@ -0,0 +1,89 @@
+package DuplicatedFunctionImplementationCheck
+
+func foobar() {}
+func foo_bar () int {}  // Compliant - has no line
```

Comment:
```suggestion
func foo_bar () {}  // Compliant - has no line
```
Otherwise there is a compilation error: missing return